testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksInsertCompatibility.java

```diff
@@ -263,6 +263,39 @@ public void testDeleteCompatibility()
         }
     }
 
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_EXCLUDE_73, PROFILE_SPECIFIC_TESTS})
+    public void testCheckConstraintsCompatibility()
```

Comment:
Does Delta OSS support https://docs.databricks.com/delta/delta-constraints.html#check-constraint ?

Reply:
Yes, that is why CI is green.

Reply:
Yes, but I omitted intentionally since the test purpose is verifying the failure and increasing the remote database coverage sounds not helpful so much. Anyway, added `DELTA_LAKE_OSS` group now.

Reply:
> The test purpose is verifying the failure and increasing the remote database coverage sounds not helpful so much.

You are right.